sonar-plugin-api/src/main/java/org/sonar/api/profiles/ProfileImporter.java

```diff
@@ -19,18 +19,19 @@
  */
 package org.sonar.api.profiles;
 
+import java.io.Reader;
 import org.apache.commons.lang.builder.ToStringBuilder;
 import org.apache.commons.lang.builder.ToStringStyle;
 import org.sonar.api.ExtensionPoint;
+import org.sonar.api.server.ComputeEngineSide;
 import org.sonar.api.server.ServerSide;
 import org.sonar.api.utils.ValidationMessages;
 
-import java.io.Reader;
-
 /**
  * @since 2.3
  */
 @ServerSide
+@ComputeEngineSide
```

Comment:
Initially added it because it's in the API.

I don't know that part, so I'll blindly trust you on this and will remove it